packages/@okta/vuepress-site/guides/custom-error-pages/use-macros/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Use Macros
+---
+The following macros contain the configuration parameters for certain page elements. These macros inject specific content or functionality automatically.
+
+#### <span v-pre>`{{orgName}}`</span> 
+Inserts the org name title.
+
+Example:
+```html
+`<title>{{orgName}} - {{errorSummary}}</title>`
```

Comment:
One too many ` here. Since you have a code block starting on line 10, no need for more `. It causes this:

![image](https://user-images.githubusercontent.com/7525482/59072063-013ffd00-8876-11e9-9844-09dea382d166.png)